src/redux/actions/standardReport.js

```diff
@@ -402,20 +403,27 @@ export const sendStandardReport = (report, isEdit) => (dispatch, getState) => {
         report.type !== reportTypes.JASPER_REPORT_TABLE
             ? omit(formattedReport, ['reportTable'])
             : formattedReport
+
+    const versionAdjustedReport = formatStandardReportPayloadForSystemVersion(
+        cleanedReport
+    )
+
```

Comment:
This is imperative code (how, not what) and imo makes feature-toggling related refactoring not harder, but more time consuming. I'd prefer if we omit variable names that hint at different versions and instead have a function that makes that process opaque, like so:

```js
const cleanNonJasperReport = report => formatStandardReportPayloadForSystemVersion(
  report.type !== reportTypes.JASPER_REPORT_TABLE
    ? omit(report, ['reportTable'])
    : report
)

const formatReport = (report, selectedReport) => cleanNonJasperReport({
  ...report,
  relativePeriods: processCheckboxValues(
      report.relativePeriods,
      selectedReport.relativePeriods
  ),
  reportParams: processCheckboxValues(
      report.reportParams,
      selectedReport.reportParams
  ),
  reportTable: report.reportTable ? { id: report.reportTable } : '',
})

// then in the standardReport.js
const formattedReport = formatReport(report)
```

This way the `standardReport.js` doesn't need to be touched, no variable name has to be changes, just the implementation of the file/function that actually has to use the toggling functionality.